src/pages/404.tsx

```diff
@@ -45,7 +42,9 @@ const SideIkkeFunnet: React.FC<{}> = () => {
                 </Heading>
                 <BodyLong spacing>The page you requested cannot be found.</BodyLong>
                 <BodyLong spacing>
-                    Go to the <Link href="https://www.nav.no/">front page</Link>, or use one of the links in the menu.
+                    Go to{" "}
+                    <Link href={`${process.env.NEXT_PUBLIC_INNSYN_ORIGIN}/sosialhjelp/innsyn`}>your applications</Link>,
```

Comment:
Holder det ikke med `href={"/"}` her?

Reply:
Vi lar det stå for da får vi lik oppførsel lokalt.